### PR TITLE
feat: Update tycho dependencies tag and re export them

### PR DESCRIPTION
**Pull request by dianacarvalho1**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -5,20 +5,14 @@
...
-//! The crate also provides optimization methods, such as golden
-//! section search, to find optimal amounts for a specific sequence.
 
 extern crate core;
 
 // Reexports
-pub use num_traits;
+pub use tycho_client;
+pub use tycho_core;
+pub use tycho_ethereum;
```
Comment (kayibal) [inline src/lib.rs:14]: Is this one required to be reexported?

Comment (dianacarvalho1) [reply src/lib.rs:14]: tycho_ethereum? huumm I'm not sure. If I'm being honest I don't even know what's in it 😅

Comment (dianacarvalho1) [reply src/lib.rs:14]: it's actually not even used in the simulation repo.. 🤷🏼‍♀️  removing it

**Review by kayibal — APPROVED** [review]
Ok looks a lot better now

**Pull request merged**